src/portal/src/app/group/add-group-modal/add-group-modal.component.ts

```diff
@@ -108,7 +110,15 @@ export class AddGroupModalComponent implements OnInit, OnDestroy {
   }
 
   resetGroup() {
-    this.group = new UserGroup();
+    this.group = new UserGroup(this.isLdapMode ? 1 : 2);
     this.groupForm.reset();
   }
+  public get isLdapMode(): boolean {
+    let appConfig = this.appConfigService.getConfig();
+    return appConfig.auth_mode === 'ldap_auth';
```

Comment:
ldap_auth and http_auth should be put in to a const. They appear more than once.